robot_controllers/src/cartesian_twist.cpp

```diff
@@ -302,15 +326,34 @@ void CartesianTwistController::command(const geometry_msgs::Twist::ConstPtr& goa
   ros::Time now(ros::Time::now());
   {
     boost::mutex::scoped_lock lock(mutex_);
+    twist_command_frame_ = frame;
     twist_command_ = twist;
     last_command_time_ = now;
   }
 
-  // Try to start up
-  if (manager_->requestStart(getName()) != 0)
+  if(frame=="")
```

Comment:
space after if (several instances below). Also, it is best to use "frame.empty()" rather than comparison